### ISSUE TITLE
Multi selection?

### ISSUE THREAD

**Issue by BarbzYHOOL** [jhawthorn/fzy#93]
I've read the readme and didn't see a multi selection option

In fzf with -m and Tab we can select multiple options

Is it available somehow or planned in fzy?

Would be useful for example for adding multiple files with git add

**Comment by marcotrosi** [comment]
yep, would be cool. and maybe a possibility to select all the files so that multi selecting all doesn't have to be selecting all one by one?

**Comment by gpanders** [comment]
I implemented this in #124. Feel free to try out my fork if/until #124 is merged.

**Comment by leo-arch** [comment]
Sadly, #124 was never merged. I implemented this myself, plus a few other features like basic color support in #170. [My fork](https://github.com/leo-arch/fzy) is also available to try it out.

**Comment by marcotrosi** [comment]
> I implemented this in #124. Feel free to try out my fork if/until #124 is merged.

still waiting for the merge 😅😘

**Comment by joereynolds** [comment]
Just in case people are unaware and come back to this thread. There is a very active fork of fzy called [fnf](https://github.com/leo-arch/fnf) which has added multi-selection and some other niceties

**Comment by stianhoiland** [comment]
I'll mention my own fork as well, www.github.com/stianhoiland/fzy. It merges most of the good PRs here (more coming, actually) and adds a couple features that I wanted after heavy use of fzy, while staying minimal.
